src/cmd/nsfs.js

```diff
@@ -4,6 +4,9 @@
 
 require('../util/dotenv').load();
 require('aws-sdk/lib/maintenance_mode_message').suppress = true;
+const cluster = /** @type {import('node:cluster').Cluster} */ (
+    /** @type {unknown} */ (require('node:cluster'))
+);
```

Comment:
yeah isn't that annoying... I fixed this in fork_utils (only type information is return correctly, it's the same object.
```suggestion
const { cluster } = require('../util/fork_utils');
);
```

Reply:
I actually did that at first, but we are not exporting it.. I saw in on other file, was it exported in the past?
this is the error I got - 
```
nsfs: exit on error TypeError: Cannot read properties of undefined (reading 'isPrimary')
    at verify_gpfslib (noobaa-core/src/cmd/nsfs.js:386:17)
    at main (noobaa-core/src/cmd/nsfs.js:315:9)
```

Reply:
maybe was removed... you can add back exports.cluster=cluster